### ISSUE TITLE
404 page missing a lang attribute

### ISSUE THREAD

**Issue by jvine** [sul-dlss/exhibits#1757]
This is pretty minor but the 404 page has no language attribute -

`<html>`
instead of
`<html lang="en">`

and it gets reported by AMP.
